equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/model/Crm14Model.java

```diff
@@ -17,4 +19,9 @@ public class Crm14Model implements CrmFileModelInterface {
     private String attachfields;
     @JsonProperty("targetpath")
     private String targetpath;
+
+    @Override
+    public CrmEvidenceFilesModel getEvidenceFiles() {
+        return (CrmEvidenceFilesModel) List.of();
+    }
 }
```

Comment:
missing new line on this file and others

Reply:
why add a new line? seems like a way of adding extra lines

Reply:
This is also a temporary solution. once the mapping is done. These should look like CRM4 instead.

Reply:
Hi, I am seeing a missing newline indicator like so on some files:-

![Screenshot 2024-06-11 at 14 32 28](https://github.com/ministryofjustice/laa-crime-equinity-historical-data/assets/1609659/2ce97a3a-052c-428b-b11d-b6b61b83465d)